UnityProject/Assets/Scripts/Items/Implants/Organs/FatProducingStomach.cs

```diff
@@ -0,0 +1,89 @@
+using Chemistry;
+using Items.Food;
+using System.Collections.Generic;
+using UnityEngine;
+using HealthV2;
+
+namespace Items.Implants.Organs
+{
+	public class FatProducingStomach : Stomach
+	{
```

Comment:
why do we have duplicated code?
this would be much simpler if the stomach just through the reagents into the bloodstream,
and then the digestive system would handle filling up the fat and spawning more as required

Reply:
Only some stomachs should produce fat though. The fat is managed by the digestive system. But this class, just means that this stomach, will produce a fat if need be.

Reply:
surely be better for the digestive system to decide if fat is generated?

Reply:
what if I have like 4 stomachs. one of them is an organic stomach that produces fat, and the others arent?

Reply:
I dont imagine a cyborg (with a human brain) would be producing fat stores. Also, having it on the stomach means different stomachs can produce different fats easily. Like right now all stomachs just make human fat, but that doesnt make sense as we develop more races.